backend/app/views/spree/admin/orders/_risk_analysis.html.erb

```diff
@@ -1,3 +1,5 @@
+<% latest_payment = @order.payments.reorder("created_at DESC").first %>
```

Comment:
what if we use `latest_payment ||=` for backward compatibility? Just in case someone is using this partial with a different local variable.

Reply:
This was done in a preparatory commit, but the final version doesn't use `latest_payment` at all. As of backwards compatibility we should be sort of good since partials don't complain about extra locals.